.github/workflows/codecoverage.yml

```diff
@@ -0,0 +1,68 @@
+---
+# This GitHub Actions workflow is calculating code coverage of extension and creates a badge for it.
+name: Code coverage
+on:
+  push:
+    branches:
+      - main
+jobs:
+  code-coverage:
+    name: "Calculate code coverage"
+    runs-on: ubuntu-18.04
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+      - name: "Install PHP"
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "${{ matrix.php-version }}"
+          coverage: xdebug
+          extensions: xdebug, mysqli
+          tools: composer:v2
+      - name: "Show Composer version"
+        run: composer --version
+      - name: "Cache dependencies installed with composer"
+        uses: actions/cache@v1
+        with:
+          key: "php${{ matrix.php-version }}-composer-${{ hashFiles('**/composer.json') }}"
+          path: ~/.cache/composer
+          restore-keys: "php${{ matrix.php-version }}-composer-\n"
+      - env:
+          TYPO3: "${{ matrix.typo3-version }}"
+        name: "Install TYPO3 Core"
+        run: |
+          composer require --no-progress typo3/minimal:"$TYPO3"
+          composer show
+      - name: "Start MySQL"
+        run: "sudo /etc/init.d/mysql start"
+      - name: "Run functional tests coverage"
+        run: |
+          export typo3DatabaseName="typo3";
+          export typo3DatabaseHost="127.0.0.1";
+          export typo3DatabaseUsername="root";
+          export typo3DatabasePassword="root";
+          composer ci:coverage:functional
+      - name: "Run unit tests coverage"
+        run: composer ci:coverage:unit
+      - name: "Merge coverage results"
+        run: composer ci:coverage:merge
+      - name: phpunit-coverage-badge
+        uses: timkrase/phpunit-coverage-badge@v1.2.0
+        with:
+          coverage_badge_path: './coverage/badge.svg'
+      - name: "Copy result to coverage directory"
+        run: cp clover.xml ./coverage/clover.xml
```

Comment:
Do we need this, or could we directly generate the merged covered `clover.xml` in `coverage/`?